lib/web/resources.go

```diff
@@ -50,7 +53,15 @@ func (h *Handler) checkAccessToRegisteredResource(w http.ResponseWriter, r *http
 		return nil, trace.Wrap(err)
 	}
 
-	resourceKinds := []string{types.KindNode, types.KindDatabaseServer, types.KindAppServer, types.KindKubeServer, types.KindWindowsDesktop}
+	resourceKinds := []string{
+		types.KindNode,
+		types.KindDatabaseServer,
+		types.KindAppServer,
+		types.KindKubeServer,
+		types.KindWindowsDesktop,
+		types.KindAppOrSAMLIdPServiceProvider,
```

Comment:
```suggestion
```

We can remove this as this does nothing. Might add `KindSAMLIdPServiceProvider` instead should that slip into `v16` in future, this will work as is. 

Also fyi that `KindAppOrSAMLIdPServiceProvider` is deprecated.

Reply:
Actually this `checkAccessToRegisteredResource` method is only kept for backward compatibility right? I think it should be fine to let it be as is since the newer web ui will hit the unified resource API anyway.